storage/migrations/16_validator_staking_history.up.sql

```diff
@@ -0,0 +1,16 @@
+BEGIN;
+
+CREATE TABLE chain.validator_balance_history
```

Comment:
TODO: Add a `db__...` test case in `common_test_cases.sh`. In e2e regressions, we only scan a tiny number of epochs, so it won't be too much data.